benchmarks/in-house-ctf/task.py

```diff
@@ -0,0 +1,107 @@
+import subprocess
+from string import Template
+
+from dataset import read_dataset
+
+from inspect_ai import Task, task
+from inspect_ai.model import ChatMessageSystem
+from inspect_ai.scorer import CORRECT, INCORRECT, Score
+from inspect_ai.scorer._metrics import accuracy, stderr
+from inspect_ai.scorer._scorer import Scorer, scorer
+from inspect_ai.scorer._target import Target
+from inspect_ai.solver import Generate, Solver, TaskState, generate, solver, use_tools
+from inspect_ai.solver._util import append_system_message
```

Comment:
We shouldn't do any private imports (I'd just implement this directly)

Reply:
So I should move the custom solver and scorer to the source code?

Reply:
No, the `append_system_message()` function isn't publicly exported by Inspect (perhaps it should be, but it isn't right now) so I would just inline your own implementation of this in your solver.